Temporarily remove references to Grafana.com (#279)
openservicemesh/osm-docs: content/docs/demos/prometheus_grafana.md

<center><i>"Save and test" should greet you with that green banner</i></center><br>

## Importing OSM Dashboards

> OSM Dashboards are available both through [our repository](https://github.com/openservicemesh/osm/tree/{{< param osm_branch >}}/charts/osm/grafana/dashboards), and are importable as json blobs through the web admin portal.

Reviewer: ```suggestion
OSM Dashboards are available through [our repository](https://github.com/openservicemesh/osm/tree/{{< param osm_branch >}}/charts/osm/grafana/dashboards), which can be imported as json blobs on the web admin portal.
```
